doc/association-rules.rst

```diff
@@ -0,0 +1,74 @@
+Association Rules
+=================
+
+.. figure:: icons/association-rules.png
+
+Induction of association rules.
+
+Signals
+-------
+
+**Inputs**:
+
+-  **Data**
+
+Data set
+
+**Outputs**:
+
+-  **Matching Data**
+
+Data instances matching the criteria.
+
+Description
+-----------
+
+This widget runs several algorithms for induction of association rules: the
+original Agrawal's algorithm for sparse data sets, a modified version by Zupan
+and Demšar, which is more suitable for the usual machine learning data, and, finally,
+an algorithm which induces classification rules where the right-hand side of the
+rule is always a class attribute.
+
+.. figure:: images/association-rules-stamped.png
+
+1. Information on the data set.
+
+2. In *Find association rules* you can set criteria for rule induction:
+
+   -  **Minimal support**: percentage of the entire data set covered
+      by the (entire) rule.
+   -  **Minimal confidence**: proportion of the number of examples which
+      fit the right side among those that fit the left side (consequent/antecedent).
+   -  **Max. number of rules**: limit to the number of rules the algorithm
+      is searching for. The default is 10000 since large data sets can slow
+      down the widget considerably.
+
+   If *Induce classification (itemset --> class) rules* is ticked, the widget
+   will look for rules that have class value on the right-hand side.
+
+   If *Auto find rules is on*, the widget will run the search at every change of
+   parameters. Might be slow for large data sets, so pressing *Find rules* only
```

Comment:
Rule generation is not really bound by the length of the dataset, rather by its width, that is, the number of distinct items (all attributes' values).